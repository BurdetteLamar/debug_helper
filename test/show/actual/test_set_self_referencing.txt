---
Set (message='test_set_self_referencing'):
  size: 1
  Element 0: 'Set #<Set: {#<Set: {...}>}>'
