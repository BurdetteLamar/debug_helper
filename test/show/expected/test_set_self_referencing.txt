---
Set (size=1 message='test_set_self_referencing'):
  Element 0: 'Set #<Set: {#<Set: {...}>}>'
